homeassistant/components/binary_sensor/insteon.py

```diff
@@ -58,7 +59,7 @@ def is_on(self):
         on_val = bool(self._insteon_device_state.value)
 
         if self._insteon_device_state.name in ['lightSensor',
-                                               'openClosedSensor']:
+                                               'ioLincSensor']:
```

Comment:
There are two devices that both have open/closed sensors inside them. The have always been thought to behave the same. Turns out they do not. The previous PR fixed the IOLinc device but broke the other one. This PR separates the behavior so that the IOLinc shows as normally open and the other stays as normally closed.